types/src/front/lib/api/assistant/agent.ts

```diff
@@ -41,7 +45,7 @@ export type AgentErrorEvent = {
 };
 
 // Event sent during the execution of an action. These are action specific.
-export type AgentActionEvent =
+export type AgentActionParamsOrOutputEvent =
```

Comment:
Maybe AgentActionSpecificEvent?

Reply:
I'm not sure I fully understand the suggestion, but I am fine with it 👍